packages/genesys-spark-components/src/components/stable/gux-icon/icon-names-deprecated.ts

```diff
@@ -1,79 +1,433 @@
+  'chat-notification-score': 'custom/message-bell-regular',
+  checkbox: 'fa/square-regular',
+  'checkbox-active': 'fa/square-check-solid',
+  'checkbox-partial': 'fa/square-minus-regular',
+  checkmark: 'custom/check-small-regular',
+  'chevron-double-down': 'fa/chevrons-down-regular',
+  'chevron-double-left': 'fa/chevrons-left-regular',
+  'chevron-double-right': 'fa/chevrons-right-regular',
+  'chevron-double-up': 'fa/chevrons-up-regular',
+  'chevron-left': 'fa/chevron-left-regular',
+  'chevron-right': 'fa/chevron-right-regular',
+  'chevron-small-down': 'custom/chevron-down-small-regular',
+  'custom/chevron-left-small-regular': 'custom/chevron-left-small-regular',
+  'chevron-small-right': 'custom/chevron-right-small-regular',
+  'chevron-small-up': 'custom/chevron-up-small-regular',
+  chrome: 'fa/chrome',
   'cjv-badge-file': 'fa/file-regular',
-  'cjv-default-event': 'cjv-custom-event',
-  'cjv-offer-closed': 'badge-x',
+  'cjv-calendar-clock': 'fa/calendar-clock-regular',
+  'cjv-custom-default': 'deprecated/cjv-custom-default',
+  'cjv-custom-event': 'fa/circle-star-solid',
+  'cjv-custom-session': 'custom/rectangules-arrow-left-to-right-regular',
+  'cjv-default-event': 'fa/circle-star-solid',
+  'cjv-delivery': 'fa/truck-regular',
+  'cjv-finance-invoice': 'fa/file-invoice-dollar-regular',
+  'cjv-form-abandoned': 'fa/file-circle-xmark-regular',
+  'cjv-form-submitted': 'custom/file-arrow-up-regular',
+  'cjv-generic-application': 'fa/file-user-regular',
+  'cjv-offer-closed': 'fa/circle-xmark-solid',
+  'cjv-offer-ignored': 'fa/subtract-circle-solid',
+  'cjv-offer-offered': 'fa/circle-chevron-right-solid',
+  'cjv-offer-opened': 'fa/badge-check-solid',
   'cjv-outcome-achieved': 'fa/circle-check-solid',
-  'cjv-page-view': 'radio-active',
-  'cjv-search': 'search',
-  'cjv-web-session': 'web',
-  'cjv-webchat-rejected': 'cjv-webchat-error',
+  'cjv-page-view': 'fa/circle-dot-regular',
+  'cjv-policy-contract': 'fa/file-contract-regular',
+  'cjv-product-added': 'fa/cart-circle-plus-regular',
+  'cjv-product-purchased': 'fa/cart-circle-check-regular',
+  'cjv-product-removed': 'fa/cart-circle-xmark-regular',
+  'cjv-search': 'fa/magnifying-glass-regular',
+  'cjv-segment': 'fa/chart-pie-simple-regular',
+  'cjv-segment-assigned': 'fa/circle-quarter-regular',
+  'cjv-web-session': 'fa/globe-regular',
+  'cjv-webchat-accepted': 'fa/message-check-solid',
+  'cjv-webchat-error': 'fa/message-xmark-solid',
+  'cjv-webchat-ignored': 'fa/message-minus-solid',
+  'cjv-webchat-offered': 'fa/message-arrow-up-right-solid',
+  'cjv-webchat-rejected': 'fa/message-xmark-solid',
+  'cjv-webchat-requested': 'fa/message-plus-solid',
+  'cjv-webchat-submitted': 'fa/message-arrow-up-solid',
+  'cjv-webchat-timeout': 'fa/timer-regular',
+  clipboard: 'fa/clipboard-regular',
+  'clipboard-error': 'fa/clipboard-question-regular',
+  'clipboard-success': 'fa/clipboard-check-regular',
+  clock: 'fa/clock-regular',
+  'clock-outline': 'fa/clock-regular',
```

Comment:
Should fa/clock-outline be removed or deprecated? It's no longer in Figma but the icon is in WCL

Reply:
This icon is not the same as the current clock-outline. So just want to double-check, this is something we want to change?